[Bug] Can't import .colpkg collection if it uses V2 Scheduler

###### Reproduction Steps

1. Settings -> Advanced -> Enable V2 Scheduler
2. Deck Pickers' top right hand menu -> Export Collection (saves as .colpkg file)
3. Deck Picker's top right-hand menu -> Import (the same file saved in Step 2)

###### Expected Result
Imports the .colpkg collection and replaces the current collection with the imported one


###### Actual Result
App crashes


###### Debug info
Log:
```
2021-08-08 22:05:46.340 1414-13301/com.google.android.providers.media.module I/MediaProvider: Deleted 1 items on external_primary due to com.ichi2.anki
2021-08-08 22:05:46.350 1414-1670/com.google.android.providers.media.module I/MediaProvider: Observed non-standard /storage/emulated/0/AnkiDroid/.nomedia
2021-08-08 22:05:46.385 310-335/? D/goldfish-address-space: claimShared: Ask to claim region [0x3cbb7b000 0x3cbf88000]
2021-08-08 22:05:46.425 310-335/? D/goldfish-address-space: claimShared: Ask to claim region [0x3cb0c3000 0x3cb4d0000]
2021-08-08 22:05:46.458 310-335/? D/goldfish-address-space: claimShared: Ask to claim region [0x3cb4d0000 0x3cb8dd000]
2021-08-08 22:05:46.502 13550-13647/com.ichi2.anki D/Utils: Finished writeToFile!
2021-08-08 22:05:46.503 13550-13647/com.ichi2.anki D/Utils: Utils.writeToFile: Size: 68 Kb, Duration: 0 s, Speed: 367 Kb/s
2021-08-08 22:05:46.509 13550-13647/com.ichi2.anki I/Utils: uncompress media
2021-08-08 22:05:46.509 13550-13647/com.ichi2.anki D/Utils: Creating new file... = /storage/emulated/0/AnkiDroid/tmpzip/media
2021-08-08 22:05:46.514 1414-1787/com.google.android.providers.media.module I/MediaProvider: Observed non-standard /storage/emulated/0/AnkiDroid/.nomedia
2021-08-08 22:05:46.535 1414-1668/com.google.android.providers.media.module I/MediaProvider: Deleted 1 items on external_primary due to com.ichi2.anki
2021-08-08 22:05:46.549 1414-1670/com.google.android.providers.media.module I/MediaProvider: Observed non-standard /storage/emulated/0/AnkiDroid/.nomedia
2021-08-08 22:05:46.563 13550-13647/com.ichi2.anki D/Utils: Finished writeToFile!
2021-08-08 22:05:46.564 13550-13647/com.ichi2.anki D/Utils: Utils.writeToFile: Size: 0 Kb, Duration: 0 s, Speed: 0 Kb/s
    
    --------- beginning of crash
2021-08-08 22:05:46.570 13550-13647/com.ichi2.anki E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #3
    Process: com.ichi2.anki, PID: 13550
    java.lang.RuntimeException: An error occurred while executing doInBackground()
        at android.os.AsyncTask$4.done(AsyncTask.java:415)
        at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
        at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
        at java.util.concurrent.FutureTask.run(FutureTask.java:271)
        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:923)
     Caused by: java.lang.AssertionError
        at com.ichi2.libanki.Storage.Collection(Storage.java:78)
        at com.ichi2.libanki.Storage.Collection(Storage.java:75)
        at com.ichi2.libanki.Storage.Collection(Storage.java:55)
        at com.ichi2.async.CollectionTask$ImportReplace.task(CollectionTask.java:1411)
        at com.ichi2.async.CollectionTask$ImportReplace.task(CollectionTask.java:1363)
        at com.ichi2.async.CollectionTask.actualDoInBackground(CollectionTask.java:198)
        at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.java:160)
        at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.java:107)
        at android.os.AsyncTask$3.call(AsyncTask.java:394)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641) 
        at java.lang.Thread.run(Thread.java:923) 
2021-08-08 22:05:46.570 13550-13647/com.ichi2.anki D/UsageAnalytics: sendAnalyticsException() description/fatal: java.lang.AssertionError/true
2021-08-08 22:05:46.574 13550-13647/com.ichi2.anki I/ACRA: ACRA is disabled for com.ichi2.anki - forwarding uncaught Exception on to default ExceptionHandler
2021-08-08 22:05:46.578 531-620/system_process W/ActivityTaskManager:   Force finishing activity com.ichi2.anki/.DeckPicker
2021-08-08 22:05:46.580 531-13654/system_process I/DropBoxManagerService: add tag=data_app_crash isTagEnabled=true flags=0x2
2021-08-08 22:05:46.584 310-335/? D/goldfish-address-space: claimShared: Ask to claim region [0x3d10b3000 0x3d1b63000]
2021-08-08 22:05:46.641 13550-13550/com.ichi2.anki D/DeckPicker: onPause()
2021-08-08 22:05:46.642 13550-13550/com.ichi2.anki I/AnkiActivity: AnkiActivity::onPause - DeckPicker

```

###### Research

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

--- a/AnkiDroid/src/main/java/com/ichi2/libanki/Storage.java
+++ b/AnkiDroid/src/main/java/com/ichi2/libanki/Storage.java
@@ -75,7 +75,7 @@ public class Storage {
         return Collection(context, path, server, log, new SystemTime());
     }
     public static Collection Collection(Context context, String path, boolean server, boolean log, @NonNull Time time) {
-        assert path.endsWith(".anki2");
+        assert (path.endsWith(".anki2") || path.endsWith(".anki21"));
         File dbFile = new File(path);
         boolean create = !dbFile.exists();
         DroidBackend backend = DroidBackendFactory.getInstance(useBackend());

